plugin.rb

```diff
@@ -30,19 +36,98 @@ class Engine < ::Rails::Engine
   end
 
   [
+    '../app/controllers/salesforce/admin_controller.rb',
+    '../app/controllers/salesforce/cases_controller.rb',
     '../app/controllers/salesforce/persons_controller.rb',
+    '../app/models/salesforce/case.rb',
     '../app/models/salesforce/person.rb',
+    '../app/serializers/concerns/case_mixin.rb',
+    '../app/serializers/case_serializer.rb',
     '../lib/salesforce/api.rb'
   ].each { |path| load File.expand_path(path, __FILE__) }
 
   Salesforce::Engine.routes.draw do
     post "/persons/create" => "persons#create"
+    post "/cases/sync" => "cases#sync"
```

Comment:
Do we have any security on these routes? I guess they should be admin only?

Reply:
Yes, I'm using `Admin::AdminController`. So it will ensure the current user is staff.